uint8[2] head
uint16 cur
uint8[10] temp
AxleCmd[18] axle_cmd